packages/api/tests/endpoints/collections/create-collection.js

```diff
@@ -180,9 +211,21 @@ test('POST creates a new collection in all data stores', async (t) => {
       version: newCollection.version,
     }
   ));
+
+  const { Messages } = await sqs().receiveMessage({
+    QueueUrl: t.context.QueueUrl,
+    WaitTimeSeconds: 10,
+  }).promise();
+
+  t.is(Messages.length, 1);
+
+  const message = JSON.parse(JSON.parse(Messages[0].Body).Message);
+  const omitFields = ['createdAt', 'updatedAt'];
+  t.is(message.event, 'Create');
+  t.deepEqual(omit(message.record, omitFields), omit(fetchedDynamoRecord, omitFields));
```

Comment:
There is no need to `omit` if we're comparing against the fetched record from dynamo which has the timestamps

Reply:
[`bdd6661` (#2432)](https://github.com/nasa/cumulus/pull/2432/commits/bdd66615c308abb672a3d4ca00c6eb92cb5eef48#diff-3e17073a9180eeafda94cabe8eaef8263b9978a4e105ba063d56185e48916081R223)